tests/test_mpi.py

```diff
@@ -12,11 +12,18 @@
                            retrieve_iteration_tree)
 from devito.mpi import MPI
 from devito.mpi.routines import HaloUpdateCall, MPICall
+from devito.mpi.distributed import CustomTopology
 from examples.seismic.acoustic import acoustic_setup
 
 pytestmark = skipif(['nompi'], whole_module=True)
 
 
+class DummyInputComm():
```

Comment:
In theory you don't even need an extra class, you can just pass `Bunch(size=...)` to CustomTopology

Reply:
Good idea, I did not know this Class